GRUB2: Disable build of documentation

ONIE does not require the building of the documentation

diff --git a/Makefile.am b/Makefile.am
index 00a9663..93e1c0e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -5,7 +5,7 @@ SUBDIRS = grub-core/gnulib .
 if COND_real_platform
 SUBDIRS += grub-core
 endif
-SUBDIRS += po docs util/bash-completion.d
+SUBDIRS += po util/bash-completion.d
 
 include $(top_srcdir)/conf/Makefile.common
 include $(top_srcdir)/conf/Makefile.extra-dist
